src/ape/api/accounts.py

```diff
@@ -192,9 +268,23 @@ def append(self, account: AccountAPI):
         self.__setitem__(account.address, account)
 
     def __setitem__(self, address: AddressType, account: AccountAPI):
+        """
```

Comment:
I don't actually see this method getting uses anywhere and am not totally sure what it's intent is.
I would ignore this one for now, maybe ask @fubuloubu

Reply:
@fubuloubu